packages/react-slider/src/components/Slider/stories/SliderControlled.stories.tsx

```diff
@@ -12,7 +12,14 @@ export const Controlled = () => {
   return (
     <>
       <Label htmlFor={id}>Control Slider [ Current Value: {sliderValue} ]</Label>
-      <Slider value={sliderValue} min={20} max={200} onChange={onSliderChange} id={id} />
+      <Slider
+        aria-valuetext={`Value is ${sliderValue}`}
```

Comment:
not necessarily a change for this PR, but in the future we should probably have a more realistic valuetext example so people don't think they need to do something like this for simple values

Reply:
what's a good example that doesn't require a bit more work. Was thinking units (degrees, dollars etc) but would probably need a bit more added to the demo for that to make sense